editor/plugins/visual_shader_editor_plugin.cpp

```diff
@@ -2971,24 +2996,30 @@ VisualShaderEditor::VisualShaderEditor() {
 	preview_shader->connect("pressed", callable_mp(this, &VisualShaderEditor::_show_preview_text));
 
 	///////////////////////////////////////
-	// PREVIEW PANEL
+	// PREVIEW WINDOW
 	///////////////////////////////////////
 
+	preview_window = memnew(Window);
+	preview_window->set_title(TTR("Resulted shader code"));
```

Comment:
I don't know if "Resulted" is correct, I think it might be better with "Resulting" or "Generated" ("Generated" is likely the clearest, provided that it's an accurate description of what this is).

Reply:
Then a button tooltip should also be changed to "Show generated shader code"?

Reply:
Alright, changed both of them..